src/qt/walletmodel.cpp

```diff
@@ -599,3 +610,7 @@ uint256 WalletModel::getLastBlockProcessed() const
 {
     return m_client_model ? m_client_model->getBestBlockHash() : uint256{};
 }
+
+CAmount WalletModel::getAvailableBalance(const CCoinControl* control) {
```

Comment:
ed0a32ec66db47b99d7d89e8c6df94b3f405f8f0, style nit:
```suggestion
CAmount WalletModel::getAvailableBalance(const CCoinControl* control)
{
```

Reply:
done